juniorguru/sync/daniel.py

```diff
@@ -47,11 +47,11 @@ async def discord_task(client):
         honza_content_size = sum(message.content_size for message in honza_messages)
 
         content = (
-            f"📊 Milý Danieli, dne {yesterday:%-d.%-m.%Y} jsi napsal {daniel_content_size} písmenek včetně mezer. "
+            f"📊 Milý Danieli, dne {yesterday:%A %-d.%-m.%Y} jsi napsal {daniel_content_size} písmenek včetně mezer. "
             f"Bylo to v {len(daniel_messages)} zprávách, v {len(daniel_channels)} různých kanálech ({len(daniel_threads)}, pokud rozlišuji vlákna). "
             f"To je jako {daniel_content_size / 1800:.1f} normostran nebo "
-            f"{daniel_content_size / 203867:.1f} dokumentů obsahujících kompletní pravidla ragby. "
             f"Honza tentýž den napsal {honza_content_size} písmenek, ale nikdy se to nedoví, protože tyhle zprávy posílám jen tobě. "
+            f"\n:abc: {daniel_content_size} :speech_left: {len(daniel_messages)} :discordthread: {len(daniel_threads)} "
```

Comment:
Custom discord emoji fungujou jinak, musí se tam dát i s IDčkem v zobáčkách, ale píšu z mobilu, tak ti neporadím (nicméně určitě někde v codebase už takové jsou)

Reply:
Je to tak a stačí `\:customemojiname:`a dostaneš potřebnou syntaxi